src/System.Collections.Concurrent/ref/System.Collections.Concurrent.builds

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Collections.Concurrent.csproj" />
+    <Project Include="System.Collections.Concurrent.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
```

Comment:
Not 100% sure but you many need to remove this netstandard1.3 build configuration as well. It will automatically be harvested from the last stable package.